packages/examples/with-vue/src/App.vue

```diff
@@ -0,0 +1,113 @@
+<script>
+import { create, insert, search as lyraSearch } from "@nearform/lyra/dist/esm/lyra";
+import { formatNanoseconds, getNanosecondsTime } from "@nearform/lyra/dist/esm/utils";
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  name: "App",
+
+  data: () => ({
+    db: null,
+    isLoading: false,
+    pokemon: [],
+    searchPokemon: [],
+    search: "",
+    elapsedTime: "",
+  }),
+
+  created() {
+    this.db = create({
+      schema: {
+        id: "number",
+        name: "string",
+      },
+    });
+  },
+
+  mounted() {
+    fetch("/pokedex.json")
+      .then(data => data.json())
+      .then(pokeList => {
+        for (const { id, name } of pokeList.pokemon) {
+          insert(this.db, {
+            id,
+            name,
+          });
+        }
+
+        this.pokemon = pokeList.pokemon;
+      })
+      .catch(console.error)
+      .finally(() => {
+        this.isLoading = false;
+      });
+  },
+
+  watch: {
+    search(newValue) {
+      if (newValue) {
+        const timeStart = getNanosecondsTime();
+
+        const result = lyraSearch(this.db, {
+          term: newValue,
+          properties: "*",
+        });
+
+        this.elapsedTime = formatNanoseconds(getNanosecondsTime() - timeStart);
+
+        const pokemons = result.hits.map(({ id: hitID }) =>
+          this.pokemon?.find(({ id: pokemonID }) => pokemonID === hitID),
+        );
+
+        if (pokemons.length && !pokemons.some(x => !x)) {
```

Comment:
```js
!pokemons.some(x => !x);
```
This got me wondering if Lyra can return `null` or `undefined` values
```js
const pokemons = [null];
```
for example 🤔

Reply:
Yep, Lyra sometimes returned undefined values (#29). That issue was fixed, so we could consider this control unnecessary but it would be better to wait for the stable version before removing it.

Reply:
Perfect!
Nice job!